packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -71,6 +78,34 @@ export async function activate(context: vscode.ExtensionContext) {
   return exportedApi;
 }
 
+async function updateProjectLineBreakpointFile() {
```

Comment:
The command that calls Apex LSP and creates a json file with the results. Decided to add it to `salesforcedx-vscode-apex` since this will also be needed for `salesforcedx-vscode-apex-debugger`.